app/pom.xml

```diff
@@ -264,7 +267,7 @@ limitations under the License.
         <dependency>
             <groupId>org.webjars</groupId>
             <artifactId>jquery</artifactId>
-            <version>3.6.4</version>
+            <version>3.7.1</version>
```

Comment:
i am always slightly worried when i update jquery since I am never sure what to look for during testing - not quite sure what is using it

Reply:
Many of the JSPs use jQuery for various things. 

You should also update the jQuery version number here: https://github.com/apache/roller/blob/af8df0aaeaf77d0fba0330a7caa348ce3b44159a/app/src/main/webapp/WEB-INF/jsps/tiles/head.jsp#L8

Reply:
it is already updated, i marked the spot below